apps/docs/content/guides/deployment/database-migrations.mdx

```diff
@@ -3,28 +3,28 @@ id: 'database-migrations'
 title: 'Database Migrations'
 description: 'How to manage schema migrations for your Supabase project.'
 subtitle: 'How to manage schema migrations for your Supabase project.'
-video: 'https://www.youtube-nocookie.com/v/vyHyYpvjaks'
-tocVideo: 'vyHyYpvjaks'
+video: 'https://www.youtube-nocookie.com/v/Kx5nHBmIxyQ'
+tocVideo: 'Kx5nHBmIxyQ'
 ---
 
-Database schema changes are managed through "migrations". Database migrations are a common way of tracking changes to your database over time.
+Database migrations are SQL statements that create, update, or delete your existing database schemas. They are a common way of tracking changes to your database over time.
```

Comment:
**nitpick**

I wonder if it would be worth for us to write a blogpost similar to: https://www.prisma.io/dataguide/types/relational/what-are-database-migrations

Explaining more in details why migrations are the standard and how to use them properly. I assume users coming to supabase would know it but maybe not.